SimG4Core/PhysicsLists/src/CMSEmStandardPhysicsEMMT.cc

```diff
@@ -0,0 +1,86 @@
+#include "G4Version.hh"
```

Comment:
This is redundant, given that it is already included in the header file

Reply:
Same as above